stac_fastapi/elasticsearch/setup.py

```diff
@@ -6,6 +6,7 @@
     desc = f.read()
 
 install_requires = [
+    "fastapi",
```

Comment:
this was used explicitly in one of the classes, but wasn't declared as a dependency (it's transitive from stac_fastapi)